ts/Series/Flags/FlagsPoint.ts

```diff
@@ -80,6 +80,14 @@ class FlagsPoint extends ColumnSeries.prototype.pointClass {
         return isNumber(this.y) || typeof this.y === 'undefined';
     }
 
+    /**
+     * @private
+     */
+    public hasNewShapeType(): (boolean|undefined) {
+        const shape = this.options.shape || this.series.options.shape;
+
+        return this.graphic && shape && shape !== this.graphic.symbolKey;
+    }
```

Comment:
This is not necessary, we [already have this on Point class](https://github.com/highcharts/highcharts/blob/8a4bd3653771624aefb1b4cf9f2d9e51bab7dac2/ts/Core/Series/Point.ts#L882-L893).

Reply:
https://github.com/highcharts/highcharts/pull/15388#discussion_r600432760